pyxnat/tests/repr_tests.py

```diff
@@ -0,0 +1,138 @@
+from pyxnat import Interface
+import os.path as op
+from . import skip_if_no_network
+
+_modulepath = op.dirname(op.abspath(__file__))
+
+fp = op.join(op.dirname(op.abspath(__file__)), 'central.cfg')
+print(fp)
+central = Interface(config=fp)
+
+proj_1 = central.select.project('surfmask_smpl')
+subj_1 = proj_1.subject('CENTRAL_S01791')
+exp_1 = subj_1.experiment('CENTRAL_E04850')
+scan_1 = exp_1.scan('11')
+resource_1 = exp_1.resource('obscure_algorithm_output')
```

Comment:
Perhaps might be a bit risky using external existing resources in Central XNAT for the CI tests, what if the owner modifies/removes them (I doubt so though)? What you think?

Reply:
Indeed, it also went through my mind. I think we may take the risk, or maybe @marinagrprat could run a `sessionmirror.py` on that experiment to make a backup copy of it in `nosetests`?

Reply:
sure!

Reply:
Session cloned into `nosetests5`, see `CENTRAL05_E02680`

Reply:
Session cloned into `nosetests5` (see `CENTRAL05_E02680`). Solved.